### ISSUE TITLE
Option to attach document to Events

### ISSUE THREAD

**Issue by ftavin** [hzi-braunschweig/SORMAS-Project#3283]
### Situation Description

Like for cases and contacts (#1771) there is a big need in France to have an option to attach documents to events.

### Feature Description

The feature should work the same way than described in #1771 but in the event part of the application.

### Additional Information

This story will use the dedicated document storage system #2328 

This issue is phase 1 : web app only (Android App out of scope)

@bernardsilenou @carolinverset

**Comment by MartinWahnschaffeSymeda** [comment]
@raghupola-vg @AndyBakcsy-she Note that in order for this to be testable a PR on SORMAS-Docker will have to be merged first and the feature activated on the test server.

See: https://github.com/hzi-braunschweig/SORMAS-Docker/pull/138
